template/model_generic.mustache

```diff
@@ -158,12 +158,12 @@ class {{classname}} {{#parentSchema}}extends {{{parent}}} {{/parentSchema}}{{^pa
      */
     public function listInvalidProperties(): array
     {
-        {{#parent}}
+        {{#parentSchema}}
```

Comment:
As I remember, you rolled back this change already

Reply:
Something went wrong when I merged from another branch and I got older change accidentally. This is the correct version